src/containers/NFT/NFTHeader/NFTHeader.tsx

```diff
@@ -48,7 +68,33 @@ const NFTHeader = (props: Props) => {
     }
   }, [tokenId])
 
-  const data = rawData && formatNFTInfo(rawData)
+  // fetch the oldest NFT transaction to get its minted data
+  const { data: firstTransaction } = useQuery(
+    ['getFirstTransaction', tokenId],
+    () => getOldestNFTTransaction(rippledSocket, tokenId),
+    {
+      enabled: !!data,
+    },
+  )
+
+  // fetch account from issuer to get the domain
+  const { data: accountData } = useQuery<AccountFormattedInfo>(
+    ['getAccountInfo'],
+    async () => {
+      const info = await getAccountInfo(rippledSocket, data?.issuer)
+      return formatAccountInfo(info, {})
+    },
+    { enabled: !!data },
+  )
+
+  const mintedDate =
+    firstTransaction && firstTransaction.transaction?.type === 'NFTokenMint'
```

Comment:
i don't see why there would ever be a `firstTransaction` but that transaction would not have a type. If that's true, you should write _either_ :

```js
const mintedDate = firstTransaction && firstTransaction.transaction.type === 'NFTokenMint' ....
```

or:

```js
firstTransaction?.transaction?.type === 'NFTokenMint' ...
```

personally I prefer the latter.